### ISSUE TITLE
Disruptive Issues Article #2 (Content)

### ISSUE THREAD

**Issue by ibportugal** [ibportugal/mms142-groupk-2023.github.io#14]

**Comment by KalmPinpin** [comment]
**What is Disruptive Technology (or a recall)**

Disruptive Technology is the term used for technology that is revolutional that it “disrupts” the normal in the technological advancement scenes. An example of disruptive technology is the internet. The things that are accessible to the people with the internet now are more and the quality of life has improved compared to the times when the internet was not so easily accessible.

**How does it impact?**
Disruptive Technology can either be improvements of old technologies or something that is totally different which makes old types of technology the latter option and take them to phasing out since the better option is the best option or the “new standard”. An example of this could be the way USB and Cloud Storage phased out Floppy Disks and CDs

**Types of Disruptive Technology**
According to an article by Repsol, some of the types of disruptive technology in the modern age are Blockchains, 5G Technology, Advanced Virtual Reality, Artificial Intelligence and Machine learning, Cloud Services, NanoTechnology and so much more. With that in mind I would like to focus my efforts in research in the field of Artificial Intelligence.


**Disruptive Technology (AI) **
Artificial Intelligence

**What is AI?**

AI or Artificial Intelligence, according to an article by Nicole Laskowski, is “the simulation of human intelligence processes by machines, especially computer systems.” Long story short it’s the emulation of a human consciousness in terms of decision making in a certain piece of technology. AI works by dumping and processing a very large amount of data and using a system to learn from the data collected. 

**What are the things that AI can do and cannot do?**

AI as it is based on the learning from the data that is processed, so far in this current age cannot understand emotional reasoning. This must be further developed as if technology is to use AI in the field that requires hospitality it would not work since it cannot properly process emotions at the current age.

**Is AI a danger or a very useful tool?**
AI is not harmful if used in the proper hands, AI can actually be helpful in predicting charts or trends based from the previous information given which can help a company or a medical field in predicting or preparing for future causes. AI can also be used to calculate things since if given the proper conditions and numbers machines are less susceptible to human error.

However, AI can be dangerous if used incorrectly or in a dangerous way. It can be used to hack or invade private information if AI systems were created to do so.


CONCLUSION
In conclusion AI is currently getting more intelligent as we speak as data is created everyday and some AI systems process more each and everyday. It’s a very powerful tool that can actually and is continually changing the world. It’s only a matter of who uses, and how they use it which determines if AI is good or bad.

**Comment by littleblackstars** [comment]
Hello, Kal-El. Kindly set your references in APA format, and modify your citation into a proper format (e.g., when was the cited website published?). Thank you!
